src/scripts/generate_relation_and_equality_ops.py

```diff
@@ -0,0 +1,271 @@
+                right_converted = int(right_value)
+            result = self.convert_true_false_to_lowercase_string(left_converted == right_converted)
+            return GenSequence.create_operation_result_object(self, '==', right_value, result)
+        return GenSequence.create_operation_result_object(self, '==', right_value, "false")
+
+    # ES v5.1 11.9.2.
+    # Not equality
+    def neq(self, left_value, right_value):
+        result = "true" if self.eq(left_value, right_value)['result'] == "false" else "true"
+        return GenSequence.create_operation_result_object(self, '!=', right_value, result)
+
+    # ES v5.1 11.9.4
+    # Strict equality
+    def seq(self, left_value, right_value):
+        # If one of the values is NaN return false
+        if (left_value == "NaN" or right_value == "NaN"):
+            return GenSequence.create_operation_result_object(self ,'===', right_value, "false")
+        result = self.convert_true_false_to_lowercase_string(left_value == right_value)
+        return GenSequence.create_operation_result_object(self, '===', right_value, result)
+
+    # ES v5.1 11.9.5.
+    # Strict not equality
+    def sneq(self, left_value, right_value):
+        result = "true" if self.seq(left_value, right_value)['result'] == "false" else "true"
+        return GenSequence.create_operation_result_object(self, '!==', right_value, result)
+
+    # Relational operators
+
+    # ES v5.1 11.8.5
+    # Less than operator
+    def less(self, left_value, right_value):
+        # If one of the values is NaN return false
+        if (left_value == "NaN" or right_value == "NaN" or
+        left_value == "undefined" or right_value == "undefined"):
+            return GenSequence.create_operation_result_object(self ,'<', right_value, "false")
+        # To convert null, true, and false
+        if left_value == "null" or left_value == "false":
+            left_converted = 0
+        elif left_value == "true":
+            left_converted = 1
+        else: # if its not null, true or false, dont do anything
+            left_converted = left_value
+        if right_value == "null" or right_value == "false":
+            right_converted = 0
+        elif right_value == "true":
+            right_converted = 1
+        else: # if its not null, true or false, dont do anything
+            right_converted = right_value
+        if (isinstance(left_converted, str) and isinstance(right_converted, str)):
+            result =  left_converted < right_converted
+        else:
+            result = int(left_converted) < int(right_converted)
+            result = self.convert_true_false_to_lowercase_string(result)
+        return GenSequence.create_operation_result_object(self, '<', right_value, result)
+
+    # ES v5.1 11.8.2
+    # Greater than operator
+    def greater(self, left_value, right_value):
+        if (left_value == "NaN" or right_value == "NaN" or
+        left_value == "undefined" or right_value == "undefined"):
```

Comment:
?
`if left_value in ["NaN", "undefined"] or right_value in ["NaN", "undefined"]:`